enhancements/installer/ipi-publishing-status.md

```diff
@@ -0,0 +1,206 @@
+---
+title: expose-cluster-publishing-status
+authors:
+  - "@flavianmissi"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@patrickdillon, installer"
+  - "@Miciah, edge networking"
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - TBD
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@bparees"
```

Comment:
Should be me or David Eads, in this case you can assign me for it

Reply:
Done